course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -11,13 +11,14 @@
             2: 'role__role_name',
             3: 'source_manual_registrar',
         };
-
+        $scope.searchInProgress = false;
```

Comment:
Can we move this down to live with the other "initializing the scope" entries after the function definitions?